src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingContext.cs

```diff
@@ -19,20 +22,20 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting
 {
     internal class FormattingContext : IDisposable
     {
-        private AdhocWorkspaceFactory _workspaceFactory;
-        private Document _csharpWorkspaceDocument;
-        private AdhocWorkspace _csharpWorkspace;
+        private readonly AdhocWorkspaceFactory _workspaceFactory;
```

Comment:
There are a couple of other random cleanups in here. My personal preference is to have a completely empty error list, including messages, so this is a step towards that.

Reply:
Heartily agree.